templates/foreman_supervisory_authority.yaml.epp

```diff
@@ -0,0 +1,25 @@
+<%- |
```

Comment:
Just thinking out loud: should we instead use a `to_yaml` method for this? We have https://github.com/theforeman/puppet-foreman/blob/master/lib/puppet/functions/foreman/to_symbolized_yaml.rb and it looks like that will give the same result.

Reply:
We merged that so you can remove this template again.

Reply:
Done.